src/pages/components/appbars/PreviewReact.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  AppBar,
+  AppBarRowMain,
+  AppBarRowHeadline,
+  AppBarLead,
+  AppBarDefault,
+  AppBarTrail
+} from "@components/AppBar/AppBar.tsx";
+
+export const PreviewReact: React.FC = () => {
+    return (
+        <AppBar rootBorder="border border-orange-500">
+            <AppBarRowMain>
```

Comment:
I think the `Main` component might be an extra level of abstraction here. Think of it like this:

![Screenshot 2024-01-05 at 4 34 39 PM](https://github.com/skeletonlabs/skeleton-next/assets/1509726/e3875cd9-8637-4fe9-9470-aa8551dc3176)

```html
<AppRail>
    <!-- Top Row -->
    <AppRailLead />
    <AppRailCenter />
    <AppRailTrail />
    <!-- Bottom Row -->
    <AppRailHeadline />
</AppRail>
```

> NOTE: left off the closing tags for simplicity here.